src/standard/x-styling.html

```diff
@@ -94,13 +94,15 @@
           var cacheableStyle = style;
           if (nativeShadow) {
             cacheableStyle = style.cloneNode ?
-            style.cloneNode(true) : Object.create(style || null);
+            style.cloneNode(true) : null;

```

Comment:
Good catch. This needs more cleanup. Previously a fake style element was returned to cache. This is no longer necessary since a style info object is cached.